neo4j/neo4j.go

```
@@ -0,0 +1,205 @@
+	}
+}
+
+// gc deletes all expired entries
+func (s *Storage) gc(t time.Time) {
+	_, _ = neo4j.ExecuteQuery(context.Background(), s.db, s.cypherGC, map[string]any{"exp": t.Unix()}, neo4j.EagerResultTransformer)
+}
+
+func mapToStruct(src map[string]any, dest any) {
+	bt, _ := json.Marshal(src)
+
+	json.Unmarshal(bt, dest)
```

golangci-lint (1.62.2)

Error return value of `json.Unmarshal` is not checked

(errcheck)